plugins/notifications-backend-module-email-notification-processor/catalog-info.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: backstage-plugin-notifications-backend-module-email-notification-processor
+  title: '@backstage/plugin-notifications-backend-module-email-notification-processor'
+  description: >-
+    The email-notification-processor backend module for the notifications
+    plugin.
+spec:
+  lifecycle: experimental
+  type: backstage-backend-plugin-module
+  owner: maintainers
```

Comment:
Not sure this is right? Should it go to the notifications project area? Will leave that up to the maintainers to determine.

Reply:
I have no idea what this file is about. @mareklibra ?